### ISSUE TITLE
start publishing ops updates to the website

### ISSUE THREAD

**Issue by georgiamoon** [m-lab/m-lab.github.io#363]

**Comment by critzo** [comment]
@nkinkade @robertodauria @georgiamoon 
We added this issue to the website repo last year, and I wonder if you have thoughts on whether we should do this or plan for a Discourse channel "operations updates / ops" or something like that instead.

**Comment by nkinkade** [comment]
@critzo: Do you mean a new public Slack channel? I don't have a strong opinion about this. I suppose I slightly lean toward thinking that publishing Ops updates on the website might be overkill. I'm fine with the existing ops@ email group, or with using some other update channels like Slack.

**Comment by robertodauria** [comment]
@critzo: I think it would be nice to have a "Status" page (mostly autogenerated and low-maintenance, leveraging our monitoring stack, something like the mlab-ns map but in a different format) where we can also publish updates - e.g. announce new/retired sites, maintenance, etc. 

Was this what you had in mind?

**Comment by critzo** [comment]
@nkinkade No, I don't think a Slack channel. We (@georgiamoon , @mehtab11 ) have been discussing options for new support infrastructure & tools -- currently researching CRM and ticketing systems, and we've discussed spinning up a Discourse site to replace the various Google groups.  I was thinking about a future, not yet deployed channel in the Discourse site. I agree that publishing ops updates to the website is more hassle than it's worth, so I'm thinking we can plan for that and leave this issue stale or close it.

**Comment by critzo** [comment]
@robertodauria I agree. I've opened a couple new issues which I think would be awesome to work on when we have time #492 #493 -- Let me know what you think. I've started work on #492 but not sure about how to do the other one.

**Comment by georgiamoon** [comment]
I like to think of our blog as a public history of activity, so I think it's important that we represent operational changes there as well. It can be something more auto-generated, or literally the same thing that we send over email, but it gives it more public access.

**Comment by critzo** [comment]
OK Sounds good. @robertodauria and @nkinkade let's just figure out a workflow that gets anything sent to ops@ to also be a blog post.
